views/landing_page.handlebars

```diff
@@ -1,21 +1,114 @@
+.welcome {
+  padding: 20px;
+  text-align: center;
+  background-color: rgb(255, 255, 255);
+  background-image: linear-gradient(#e6b2b2,rgb(247, 246, 187),rgb(247, 246, 187),rgb(214, 247, 187),rgb(214, 247, 187), white);
+  color: black;
+  font-size: 30px;
+  position: fixed;
+  top: 6.5%;
+  left: -4%;
+  width: 100%;
+}
+.add_user{
+  border-radius: 5px;
+  background-color: #e6b2b2;
+  padding: 20px;
+  text-align: center;
+ 
+
+  box-shadow: 0 0.4px 0.4px rgba(128, 128, 128, 0.109),
+    0 1px 1px rgba(128, 128, 128, 0.155),
+    0 2.1px 2.1px rgba(128, 128, 128, 0.195),
+    0 4.4px 4.4px rgba(128, 128, 128, 0.241),
+    0 12px 12px rgba(128, 128, 128, 0.35);
+}
+.new_project_board{
+  border-radius: 5px;
+  background-color: rgb(247, 246, 187);
+  padding: 20px;
+  text-align: center;
+
+
+  box-shadow: 0 0.4px 0.4px rgba(128, 128, 128, 0.109),
+    0 1px 1px rgba(128, 128, 128, 0.155),
+    0 2.1px 2.1px rgba(128, 128, 128, 0.195),
+    0 4.4px 4.4px rgba(128, 128, 128, 0.241),
+    0 12px 12px rgba(128, 128, 128, 0.35);
+}
+.project_button{
+  border-radius: 5px;
+  background-color: rgb(214, 247, 187);
+  padding: 20px;
+  text-align: center;
+
+  box-shadow: 0 0.4px 0.4px rgba(128, 128, 128, 0.109),
+    0 1px 1px rgba(128, 128, 128, 0.155),
+    0 2.1px 2.1px rgba(128, 128, 128, 0.195),
+    0 4.4px 4.4px rgba(128, 128, 128, 0.241),
+    0 12px 12px rgba(128, 128, 128, 0.35);
+}}
+ </style>
+</head>
+
+<body>
+    <main>
+    <div class="welcome">
+      <h1>Welcome to Leyamo Kanban</h1>
+    </div>
+    
     <aricle>
-        <form action="/add_user" method="POST">
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+      <br>
+        <form class="add_user" 
+          action="/add_user" method="POST">
             <label>Create a new user</label>
             <input name="name" type="text" placeholder="User's name" required/>
             <br>
             <label>Image URL</label>
             <input name="image" type="url" placeholder="Image URL" required/>
             <br>
             <button>Submit</button>
+            <br>
         </form>
-        <form action="/new_project_board" method="POST">
+           <br>
+           
+          <form class="new_project_board"
+           action="/new_project_board" method="POST">
             <label>Create a new project board</label>
             <input name="name" type="text" placeholder="Project board's name" required/>
             <br>
             <button>Submit</button>
-        </form>
-        <a href="/view_all_projects"><button>View All Project Boards</button></a>
+            <br>
+          </form>
+            <br>
+
+         <form class="project_button"
+           <a href="/view_all_projects">
+            <label>Already have a Project Board?</label>
+            <br>
+            <button>View All Project Boards</button>
+            <br>
+          </form>
+            <br>
+    
+
+        
     </aricle>
```

Comment:
<article> missing a t, not sure if it matters too much!

Reply:
oh thank you. I don't think it does but I will fix that anyway